app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -52,31 +57,20 @@ public void onCreate(Bundle savedInstanceState) {
         speakersRecyclerView = (RecyclerView) findViewById(R.id.list_speakerss);
         List<Speaker> speakers = null;
 
-        try {
-            speakers = dbSingleton.getSpeakersbySessionName(title);
-            session = dbSingleton.getSessionbySessionname(title);
-        } catch (ParseException e) {
-            e.printStackTrace();
-        }
-        try {
-            tv_room1.setText((dbSingleton.getMicrolocationById(session.getMicrolocations())).getName());
-        } catch (ParseException e) {
-            e.printStackTrace();
-        }
+        speakers = dbSingleton.getSpeakersbySessionName(title);
+        session = dbSingleton.getSessionbySessionname(title);
+
+        tv_room1.setText((dbSingleton.getMicrolocationById(session.getMicrolocations())).getName());
```

Comment:
why is this text view  called tv_room1 ? I don't see underscores being used in Java anywhere. This is c++ style. Also tv == TV (television) or what? Incoherent.